bucket/restic.json

```
@@ -1,5 +1,6 @@
 {
     "homepage": "https://restic.github.io/",
```

There is static redirect.

```suggestion
    "homepage": "https://restic.net/",
```